lib/ziti_tunneler.c

```diff
@@ -19,6 +19,10 @@ limitations under the License.
 #define LWIP_DONT_PROVIDE_BYTEORDER_FUNCTIONS 1
 #endif
 
+#if defined(__mips) || defined(__mips__)
```

Comment:
I'm curious why this is being ifdef'ed, since it was being defined on all architectures before this branch.

Reply:
we never build for mips before, I think it could be because of the older gcc that have to be used